pkg/acs/calacs/lib/mkspt.c

```diff
@@ -167,53 +167,48 @@ int mkNewSpt (char *in_list, char *mtype, char *output) {
                 trlmessage (MsgText);
 		    }
 
-	        /* Update the FILENAME header keyword */
-            if (PutKeyStr (&header, "FILENAME", out_spt, "")) {
-                reportErrorAndFreeHeader (&header);
```

Comment:
You've removed the error but why have you also removed the free? Don't this still leak now?